pkg/pillar/dpcreconciler/linux.go

```diff
@@ -1783,7 +1783,7 @@ func (r *LinuxDpcReconciler) getIntendedFilterRules(gcp types.ConfigItemValueMap
 	// Allow all ICMP traffic to enter the device from outside.
 	icmpRule := iptables.Rule{
 		RuleLabel:   "Allow ICMP",
-		MatchOpts:   []string{"-p", "icmp"},
+		MatchOpts:   []string{"-p", "icmp", "--icmp-type", "echo-request"},
 		Target:      "ACCEPT",
 		Description: "Allow ICMP traffic to enter the device from outside",
```

Comment:
LGTM and eden tests are green, I would just suggest to also update the `RuleLabel` and `Description` to something like `Allow ICMP ping` and `Allow ICMP echo request to enter the device from outside`, respectively.